openrtb/bidresponse.go

```diff
@@ -22,7 +24,6 @@ type BidResponse struct {
 	NoBidReason NoBidReason `json:"nbr,omitempty"`
 
 	RawExtension json.RawMessage `json:"ext,omitempty"`
-	Bundle       string          `json:"bundle,omitempty"`
 	Extension    interface{}     `json:"-"` // Opaque value that can be used to store unmarshaled value in ext field.
```

Comment:
RawExtension and Extension are confused.

Reply:
Yes, but Jaeger depends on this confusing design. If anyone can explain Jaeger's related code, we can fix it.